warehouse/utils/utils.go

```diff
@@ -519,3 +519,13 @@ func IdentityMappingsUniqueMappingConstraintName(warehouse WarehouseT) string {
 func GetWarehouseIdentifier(destType string, sourceID string, destinationID string) string {
 	return fmt.Sprintf("%s:%s:%s", destType, sourceID, destinationID)
 }
+
+func DoubleQuoteColumnNamesAndJoinByComma(keys []string) (quotedColumnNames string) {
```

Comment:
Let's rename this function? This function works for any strings and need not be column names.

Reply:
`DoubleQuoteAndJoinByComma `